demo-terraform/aws/wordpress_ec2_alb_rds/vpc_flows.tf

```diff
@@ -1,23 +1,25 @@
 
 locals {
-    log_name = "flows-${random_string.lab_id.result}"
-    role_name = "role-${random_string.lab_id.result}"
-    rolepol_name = "rolepol-${random_string.lab_id.result}"
+  log_name     = "flows-${random_string.lab_id.result}"
+  role_name    = "role-${random_string.lab_id.result}"
+  rolepol_name = "rolepol-${random_string.lab_id.result}"
 
 }
 
 resource "aws_flow_log" "gc_lab" {
-    iam_role_arn    = aws_iam_role.vpc_flows_role.arn
-    log_destination = aws_cloudwatch_log_group.vpc_flows.arn
-    traffic_type = "ALL"
-    log_format = var.vpc_flow_log_format
-    vpc_id          = module.vpc.vpc_id
-    max_aggregation_interval = 60
+  iam_role_arn             = aws_iam_role.vpc_flows_role.arn
+  log_destination          = aws_cloudwatch_log_group.vpc_flows.arn
+  traffic_type             = "ALL"
+  log_format               = var.vpc_flow_log_format
+  vpc_id                   = module.vpc.vpc_id
+  max_aggregation_interval = 60
 }
 
 resource "aws_cloudwatch_log_group" "vpc_flows" {
```

Comment:
**Accurics remediation**
 
**[HIGH]: Ensure KMS CMK is used to encrypt AWS CloudWatch Log Group**
 
**Vulnerability**
KMS CMK is not used to encrypt AWS CloudWatch Log Group which may lead to sensitive data exposure.
 
**Remediation**
AWS CloudWatch Log Group can be encrypted using AWS KMS.
 In AWS Console - 
 1. Create an AWS KMS customer managed key. 
 2. Set the necessary permissions on the customer managed key. 
 3. Associate a log group with a customer managed key.

source: accurics-